lib/Jsrt/JsrtCommonExports.inc

```diff
@@ -103,6 +103,8 @@
     JsGetRuntime
     JsIdle
     JsSetPromiseContinuationCallback
+    JsCreateWeakReference
+    JsGetWeakReferenceValue
```

Comment:
Since you moved the headers to `ChakraCore.h`, you should also move these lines below the `#ifndef` below as well, I believe.